proxy/pkg/user/backend/cs3.go

```diff
@@ -9,7 +9,6 @@ import (
 	rpcv1beta1 "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
 	types "github.com/cs3org/go-cs3apis/cs3/types/v1beta1"
 	"github.com/owncloud/ocis/ocis-pkg/log"
-	"github.com/owncloud/ocis/ocis-pkg/oidc"
```

Comment:
- [ ] actually the `oidc.StandardClaims` may no longer be necessary at all

Reply:
The ocis-pkg oidc middleware uses it ... I doubt we currently need that ... kill it with 🔥 

Currently, the proxy authenticates requests and mints an internal JWT that the other services rely on. Changing this would be an architectural change with advantages and disadvantages. Something to be discussed in an ADR... maybe someday.